resolver/src/test/java/org/apache/james/jspf/core/SPFSessionTest.java

```diff
@@ -19,19 +19,53 @@
 
 package org.apache.james.jspf.core;
 
-import org.apache.james.jspf.core.exceptions.NoneException;
-import org.apache.james.jspf.core.exceptions.PermErrorException;
+import org.junit.Test;
 
-import junit.framework.TestCase;
+import static org.junit.Assert.assertEquals;
 
-public class SPFSessionTest extends TestCase {
-
-    /*
-     * Test method for 'org.apache.james.jspf.core.SPF1Data.getMacroIpAddress()'
-     */
-    public void testGetMacroIpAddress() throws PermErrorException, NoneException {
+public class SPFSessionTest {
+    @Test
+    public void testGetMacroIpAddress() {
         SPFSession d = new SPFSession("mailfrom@fromdomain.com","helodomain.com","2001:DB8::CB01");
+        System.out.println(d.getMacroIpAddress());
         assertEquals("2.0.0.1.0.D.B.8.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.C.B.0.1",d.getMacroIpAddress());
     }
 
+    @Test
+    public void shouldReturnInvalidIP6() {
+        String addr = "2001:db8::644:f:f:f:x";
+        SPFSession spfSession = new SPFSession("", "", addr);
+        System.out.println(spfSession.getInAddress());
```

Comment:
Maybe get rid of the intermediate System.out.println ?

Reply:
Sure, I will remove. I forgot this println I added when I was testing the getInAddress.